docgen/src/guide/Multi_index.md

```diff
@@ -6,22 +6,19 @@ category: guide
 navWeight: 50
 ---
 
-You can use multiple [`<InstantSearch>`](guide/<InstantSearch>.html) instances for cases like:
+Whenever you want to:
 
-* displaying hits from different indices
-* sharing a single SearchBox
-* any use case involving synchronizing widgets between different [`<InstantSearch>`](guide/<InstantSearch>.html) instances
+* display hits from different indices
+* share a single SearchBox
+* synchronize widgets between different indices
 
-Two props on the [InstantSearch component](widgets/InstantSearch.html) can be used to inject searchState or be notified of searchState changes:
+You can use multiple [`<MultiIndexContext>`](widgets/<MultiIndexContext>.html) instances.
```

Comment:
I don't really like the `MultiIndexContext` as a name. Can a better one be searched?

Maybe just `<Index />`, since the use case is multiple indices, where each of these components implies a separate index? How were you thinking of solving this in vue @rayrutjes